examples/trials/network-morphism/config.yml

```diff
@@ -0,0 +1,25 @@
+authorName: default
+experimentName: example_cifar10-network-morphism
+trialConcurrency: 4
+maxExecDuration: 48h
+maxTrialNum: 200
+#choice: local, remote, pai
+trainingServicePlatform: local
+#searchSpacePath: search_space.json
+#choice: true, false
+useAnnotation: false
+tuner:
+  #choice: TPE, Random, Anneal, Evolution, BatchTuner, Medianstop, NetworkMorphism
```

Comment:
Medianstop is a tuner????? please remove this.

Reply:
thx.